### PR TITLE
Preparing to remove "geog_coverage"

### PR DESCRIPTION
**Pull request by avdata99**
Related to #735 

This PR:
 - Creates a command to move any field to packages `notes`
 - If some `geog_coverage` was moved to notes, the new `geographies` field show it.

After running the command, in the dataset form

image

image

### CODE REVIEW

File: ckanext/unhcr/click_commands.py [hunk 1]
```diff
@@ -202,3 +202,59 @@ def expire_users(verbose):
...
+    )
+
+    click.secho('{} datasets found'.format(packages['count']), bold=True)
+
+    updated = 0
+    ignored = 0
+    field_label = field_label or field_name
+
+    for pkg in packages['results']:
+        to_notes = '\n{}: {}\n'.format(field_label, pkg[field_name])
```
Comment (amercader) [inline ckanext/unhcr/click_commands.py:240]: Can you add a couple more line breaks between the existing notes and the new label?

File: ckanext/unhcr/templates/scheming/form_snippets/geographies.html [hunk 2]
```diff
@@ -1,5 +1,14 @@
 <div class="form-group control-full">
   <label class="control-label" for="field-geographies">Geographies</label>
+  {% set removed_geog = h.get_removed_geog_coverage(data['notes']) %}
+  {% if removed_geog %}
+  <br/>
+  <span style="color:red">
+    Originally, this dataset used <b>"{{ removed_geog }}"</b> as Geographic Coverage.
+    You can use it here now and remove this reference in the dataset notes
```
Comment (amercader) [inline ckanext/unhcr/templates/scheming/form_snippets/geographies.html:8]:
```suggestion
    This dataset used <b>"{{ removed_geog }}"</b> in the old free-text Geographic Coverage field. You can add the equivalent structured coverage below and remove the reference in the dataset notes.
```

File: ckanext/unhcr/templates/scheming/form_snippets/geographies.html [hunk 3]
```diff
@@ -1,5 +1,14 @@
 <div class="form-group control-full">
   <label class="control-label" for="field-geographies">Geographies</label>
+  {% set removed_geog = h.get_removed_geog_coverage(data['notes']) %}
+  {% if removed_geog %}
+  <br/>
+  <span style="color:red">
```
Comment (amercader) [inline ckanext/unhcr/templates/scheming/form_snippets/geographies.html:6]: Can you use a `<div class="alert warning">` here?

**Review by amercader — APPROVED** [review]

**Comment by amercader** [comment]
@avdata99 can you run commands on the uat and prod containers? if not easy you might want to wrap the command in an action for easy API access

**Comment by avdata99** [comment]
Closing/archiving this because we are not removing the field for now

**Pull request closed without merging**